SU-SWS/stanford_basic
# DCORE-457: Misc updates

# READY FOR REVIEW

# Summary
- Update block positions
- Add template for page title fields
- Remove duplicated components that were moved to jumpstart_ui

# Needed By (Date)
- Soonish

# Urgency
- Not ultimately

# Steps to Test

1. Review code
2. Check out this branch and install stanford_profile
3. Edit a content type with layout builder and place the title block somewhere
4. Validate title block shows up as H1
5. Validate that removed components are still available through jumpstart_ui module

# Affected Projects or Products
- ?

# Associated Issues and/or People
- D8CORE-457

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
